src/mo_interpreter/interpret.ml

```diff
@@ -421,8 +421,36 @@ and interpret_exp_mut env exp (k : V.value V.cont) =
     k (interpret_lit env lit)
   | ActorUrlE url ->
     interpret_exp env url (fun v1 ->
-      match Ic.Url.decode_principal (V.as_text v1) with
-      | Ok bytes -> k (V.Blob bytes)
```

Comment:
OK, no wonder lots of test are failing. This used to return a blob, but is now returning an object, so the tests will fail in different ways.

Fundamentally, the problem is that the frontend interpreter needs to model actor id and represent actors values using actor ids, not objects.